packages/icons/CONTRIBUTING.md

```diff
@@ -0,0 +1,50 @@
+# Cockroach Icons
```

Comment:
I have never seen contributing files that aren't in the project repository's root, `docs`, or `.github` folder. I think all of this information should go in the readme. This is related to the conversation around what a contributing doc is for vs what a readme is for: https://github.com/cockroachdb/ui/issues/36#issuecomment-636073556